### ISSUE TITLE
Typo in Documentation?

### ISSUE THREAD

**Issue by ginward** [cornellius-gp/gpytorch#1447]
I am not sure if there is a typo in the [documentation](https://docs.gpytorch.ai/en/stable/examples/05_Deep_Gaussian_Processes/Deep_Gaussian_Processes.html?highlight=nll): 

```
class DeepGP(DeepGP):
    def __init__(self, train_x_shape):
        hidden_layer = ToyDeepGPHiddenLayer(
            input_dims=train_x_shape[-1],
            output_dims=num_output_dims,
            mean_type='linear',
        )

        last_layer = ToyDeepGPHiddenLayer(
            input_dims=hidden_layer.output_dims,
            output_dims=None,
            mean_type='constant',
        )

        super().__init__()

        self.hidden_layer = hidden_layer
        self.last_layer = last_layer
        self.likelihood = GaussianLikelihood()

    def forward(self, inputs):
        hidden_rep1 = self.hidden_layer(inputs)
        output = self.last_layer(hidden_rep1)
        return output

    def predict(self, test_loader):
        with torch.no_grad():
            mus = []
            variances = []
            lls = []
            for x_batch, y_batch in test_loader:
                preds = model.likelihood(model(x_batch))
                mus.append(preds.mean)
                variances.append(preds.variance)
                lls.append(model.likelihood.log_marginal(y_batch, model(x_batch)))

        return torch.cat(mus, dim=-1), torch.cat(variances, dim=-1), torch.cat(lls, dim=-1)
```

Should the predict function be using self, instead of model? 
```
    def predict(self, test_loader):
        with torch.no_grad():
            mus = []
            variances = []
            lls = []
            for x_batch, y_batch in test_loader:
                preds = self.likelihood(self(x_batch))
                mus.append(preds.mean)
                variances.append(preds.variance)
                lls.append(self.likelihood.log_marginal(y_batch, self(x_batch)))
```

I might be wrong, as my knowledge of the package is limited. But I think I might just point it out for the sake of my own understanding :)

Thanks!

**Comment by gpleiss** [comment]
Yes this is a typo. Could you put up a PR to fix it?

**Comment by ginward** [comment]
> Yes this is a typo. Could you put up a PR to fix it?

Thanks! I have created the pull request. Please review and let me know if it works.

**Comment by gpleiss** [comment]
Closed by #1473
